bauhaus/core.py

```diff
@@ -525,6 +525,23 @@ def foo(self):
     # Constraint creation for these are directed to
     # constraint._constraint_by_function.
 
+    def none_of(encoding: Encoding, **kwargs):
```

Comment:
move this function to above the comments on line 524/directly after the implies all function!